packages/cli/src/commands/spaces/vpn/wait.ts

```diff
@@ -11,17 +12,35 @@ const wait = (ms: number) => new Promise(resolve => {
 export default class Wait extends Command {
     static topic = 'spaces';
     static description = 'wait for VPN Connection to be created';
+    static examples = [heredoc`
+      $ heroku spaces:vpn:wait vpn-connection-name --space my-space
+      Waiting for VPN Connection vpn-connection-name to allocate... done
+      === my-space VPN Tunnels
+
+     VPN Tunnel Customer Gateway VPN Gateway    Pre-shared Key Routable Subnets IKE Version
+     ────────── ──────────────── ────────────── ────────────── ──────────────── ───────────
+     Tunnel 1    104.196.121.200   35.171.237.136  abcdef12345     10.0.0.0/16       1
+     Tunnel 2    104.196.121.200   52.44.7.216     fedcba54321     10.0.0.0/16       1
+    `]
+
     static flags = {
       space: flags.string({char: 's', description: 'space the vpn connection belongs to', required: true}),
-      name: flags.string({char: 'n', description: 'name or id of the vpn connection to wait for', required: true}),
       json: flags.boolean({description: 'output in json format'}),
       interval: flags.string({char: 'i', description: 'seconds to wait between poll intervals'}),
       timeout: flags.string({char: 't', description: 'maximum number of seconds to wait'}),
-    };
+    }
+
+    static args = {
+      name: Args.string({
+        description: 'name or id of the VPN connection to get info from',
```

Comment:
Here too. I suggest changing description to something like `name or id of the VPN connection you'll be waiting on for allocation.`